src/styles/style.css

```diff
@@ -0,0 +1,16 @@
+@import url(./card-block.css);
+@import url(./stars-block.css);
+@import url(./normalize.css);
+
+:root {
+  --white: #fff;
+  --colorDark: #060b35;
+  --colorLight: #616070;
+  --btncolor: #00acdc;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  background-color: var(--colorDark);
+}
```

Comment:
Will be better to move this part to normalize.css

Reply:
agree, check this comment
will be better to move these styles to `normalize.css`